galsim/phase_psf.py

```diff
@@ -1068,15 +1080,18 @@ def __str__(self):
 
     def __repr__(self):
         outstr = ("galsim.PhaseScreenPSF(%r, lam=%r, exptime=%r, flux=%r, aper=%r, theta=%r, " +
-                  "scale_unit=%r, interpolant=%r, gsparams=%r)")
+                  "interpolant=%r, scale_unit=%r, gsparams=%r)")
```

Comment:
It looks like you've swapped scale unit  and interpolant here, but not in the line below, so they are now inconsistent?